Vagrantfile

```diff
@@ -19,6 +19,11 @@ Vagrant.configure("2") do |config|
   end
   config.vm.provision "shell", path: "bootstrap.sh", privileged: false
 	
+	
+config.vm.network(
+"forwarded_port", guest: 8000, host: 8000, host_ip: "127.0.0.1"
+)
+
```

Comment:
I think those lines should be with double space, that way it will be in the do - end block.

Reply:
I changed it.